docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -192,3 +192,36 @@ keep the Password field empty:
 ![DataGrip connection options](../../../img/database-access/guides/sqlserver/datagrip-connection@2x.png)
 
 Click OK to connect.
+
+## Redis Insight
+
+<Admonition type="note">
+  Teleport's Redis Insight integration only supports Redis standalone instances.
+</Admonition>
+
+After opening Redis Insight click `ADD REDIS DATABASE`.
+
+![Redis Insight Startup Screen](../../../img/database-access/guides/redis/redisinsight-startup.png)
+
+Click `Add Database Manually`. Use `127.0.0.1` as the `Host` and port printed by `tsh proxy db`.
```

Comment:
I'm not sure what the second sentence means. Does `tsh proxy db` return a port to include in a Redis Insight field? If so, we should change this to:

```markdown
Run the following command to start a proxy that you can use to connect
to your Redis instance:

\`\`\`code (remove the escapes)
$ tsh proxy db
\`\`\`

Copy the port returned from this command and use it for the `Port`
field. Use `127.0.0.1` for the `Host` field.

Reply:
This is not that simple.  First you need to login to the database. Then call the `tsh db proxy`. This is explained at the top of the page https://goteleport.com/docs/database-access/guides/gui-clients/#get-connection-information. I don't think it's needed to repeat it here. I can add a link to the top of that page to make it more clear.